src/content/en/updates/2020/08/devtools.md

```diff
@@ -0,0 +1,261 @@
+
+Prior to the new media panel in DevTools, logging and debug information about video players could be found in [chrome://media-internals](chrome://media-internals).
+
+The new Media panel provides an easier way to view events, logs, properties, and a timeline of frame decodes in the same browser tab as the video player itself. You can live view and inspect on potential issues quicker (e.g. why dropped frames are occurring, why JavaScript is interacting with the player in an unexpected way).
+
+For engineers who are constantly testing video playback quality, working on a JavaScript front-end to a media player, or working on re-encoding tools, this new Media panel could greatly increase your productivity.
+
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/bf98fea787a8c2db4e0763ae2095faf8112458eb #}
+
+Chromium issues: [1018414](https://crbug.com/1018414)
+
+
+## Issues tab updates {: #issues-tab }
+
+The Issues warning bar on the Console panel is now replaced with a regular message. 
+
+![Issues in console message](/web/updates/images/2020/08/issue-console-msg.png)
+
+Third party cookie issues are now hidden by default in the Issues tab. Tick the newly added “Include third-party cookie issues” checkbox to view them.
+
+![thrid party cookie issues checkbox](/web/updates/images/2020/08/third-party-cookies.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/bb660b996589727edf16be103343f91d38b571fa #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/dea8897a72d4954ec04723bb0bbdf5c90440a0f1 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/760ee9c0d70d6bce108532c164fbdbc5cb2c2951 #}
+
+Chromium issues: [1096481](https://crbug.com/1096481), [1068116](https://crbug.com/1068116), [1080589](https://crbug.com/1080589)
+
+
+## Emulate missing local fonts {: #emulate-local-fonts }
+
+Open the [Rendering tab](/web/tools/chrome-devtools/evaluate-performance/reference#rendering) and use the new “Disable local fonts” feature to emulate missing `local() sources in @font-face rules.
+
+For example, when the font “Rubik” is installed on your device and the `@font-face src` rule uses it as a  `local()` font, Chrome uses the local font file from your device.
+
+When “Disable local fonts” is ticked, DevTools ignores the `local()` fonts and fetches them from the network.
+
+![Emulate missing local fonts](/web/updates/images/2020/08/disable-font.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/5b6a769ee19c084fdda6482d2d8f1ce9f7e37ac0  #}
+
+Chromium issues: [384968](https://crbug.com/384968)
+
+
+## Emulate inactive users {: #emulate-inactive-users }
+
+The [Idle Detection API](https://web.dev/idle-detection) allows developers to detect inactive users and react on idle state changes. You can now use DevTools to emulate idle state changes in the Sensors tab for both the user state and the screen state instead of waiting for the actual idle state to change.
```

Comment:
```suggestion
The [Idle Detection API](https://web.dev/idle-detection) allows developers to detect inactive users and react on idle state changes. You can now use DevTools to emulate idle state changes in the **Sensors** tab for both the user state and the screen state instead of waiting for the actual idle state to change. You can open the **Sensors** tab from the [**Drawer**](/web/tools/chrome-devtools/customize/#drawer).
```